### ISSUE TITLE
Offset / Pad Keyboard from bottom

### ISSUE THREAD

**Issue by AzzyC** [openboard-team/openboard#630]
**Is your feature request related to a problem? Please describe.**
As displays on newer phones are becoming stretched, it is less feasible to hold the phone from the bottom making it difficult to reach keys close to the Gesture / Navigation Bar

**Describe the solution you'd like**
A feature to offset or pad the keyboard from the bottom of the screen to improve keystrokes

**Describe alternatives you've considered**
FlorisBoard has this feature incorporated

**Additional context**
Please find attached screenshots showing the opportunity of padding the keyboard:
![IMG_20220630_144151](https://user-images.githubusercontent.com/36119634/176694700-1b3ec566-aa55-41eb-9af1-bfdf8c4805bc.jpg)
![IMG_20220630_144207](https://user-images.githubusercontent.com/36119634/176694708-17af576d-ae73-47de-955d-71f02655c3c2.jpg)

**Comment by frithrah** [comment]
Yes. I find it hard to press the spacebar consistently when using OpenBoard on a tablet. Adding an offset from the bottom would help tremendously.
